apis/swagger.yml

```diff
@@ -567,6 +581,33 @@ definitions:
       message:
         type: string
 
+  DragonflyVersion:
+    type: "object"
+    description: |
+      Version and build information from supernode.
+    properties:
+      version:
+        type: "string"
+        description: "Version of supernode"
+      revision:
+        type: "string"
+        description: "Git commit when building supernode"
+      branch:
```

Comment:
I am wondering if we need this branch field. commit ID is uniquely to the commit history. Branch is redundant. Some users will directly use the source code to build supernode binary, while there is no branch. Or branch mean nothing, since I can build commit A from branch and I can also build commitB from branch.

Reply:
Make sense.